honeybadger/core.py

```diff
@@ -11,10 +11,10 @@
 import honeybadger.connection as connection
 import honeybadger.fake_connection as fake_connection
 from .events_worker import EventsWorker
-from .payload import create_payload
 from .config import Configuration
+from .notice import Notice
 
-logging.getLogger("honeybadger").addHandler(logging.NullHandler())
+logger = logging.getLogger("honeybadger")
```

Comment:
I _believe_ removing the null handler will attach the logger to the root.

https://github.com/honeybadger-io/honeybadger-python?tab=readme-ov-file#logging